### PR TITLE
Small php issue

### PR DESCRIPTION
**Pull request by sohelrana820**
Fixed some small PHP issues.

### CODE REVIEW

File: tests/TestCase/Filesystem/FileTest.php [hunk 1]
```diff
@@ -533,7 +533,7 @@ public function testWrite()
     /**
      * testAppend method
      *
-     * @return void
+     * @return boolean|void
```
Comment (dereuromark) [inline tests/TestCase/Filesystem/FileTest.php]: The test should rather be fixed as this should be void.

File: src/Http/Client.php [hunk 2]
```diff
@@ -179,7 +179,7 @@ public function __construct($config = [])
     /**
      * Get the cookies stored in the Client.
      *
-     * @return \Cake\Http\Client\CookieCollection
+     * @return CookieCollection
```
Comment (dereuromark) [inline src/Http/Client.php]: why?

Comment (sohelrana820) [reply src/Http/Client.php]: Cause $this->_cookies is a object of Cake\Http\Cookie\CookieCollection. But it was return the object of \Cake\Http\Client\CookieCollection

Comment (dereuromark) [reply src/Http/Client.php]: But please fix it to the FQCN as it was before, just the right one.

Comment (ADmad) [reply src/Http/Client.php]: Then you need to use the correct FQCN not just `CookieCollection`.

Comment (sohelrana820) [reply src/Http/Client.php]: OK, then I am reverting this changes.

Comment (dereuromark) [reply src/Http/Client.php]: Just fix it to the expected `\Cake\Http\Cookie\CookieCollection` then.

Comment (sohelrana820) [reply src/Http/Client.php]: Just want to ensure... After fixing this I need to send a new pull request, right?

Comment (dereuromark) [reply src/Http/Client.php]: No, you can add a 2nd commit on top of this one.

File: src/Console/ConsoleInput.php [hunk 3]
```diff
@@ -77,7 +77,8 @@ public function read()
     public function dataAvailable($timeout = 0)
     {
         $readFds = [$this->_input];
-        $readyFds = stream_select($readFds, null, null, $timeout);
+        $null = null;
```
Comment (dereuromark) [inline src/Console/ConsoleInput.php]: What is the value of this? The repeating null is acceptable in the case it was here.

File: tests/test_app/TestApp/Controller/CakesController.php [hunk 4]
```diff
@@ -39,7 +39,7 @@ public function noRender()
     /**
      * invalid method
      *
-     * @return \Cake\Http\Response
+     * @return string
```
Comment (dereuromark) [inline tests/test_app/TestApp/Controller/CakesController.php]: The point of this kind of is to show that this is an invalid and undocumented return type.
Not sure if we should switch the documentation here.

File: src/Console/ConsoleInput.php (hunk 3 above)
Comment (ADmad) [reply src/Console/ConsoleInput.php]: Please revert this unnecessary change.

Comment (sohelrana820) [reply src/Console/ConsoleInput.php]: stream_select() expect reference value. And null can't be passed by a reference.

Comment (ADmad) [reply src/Console/ConsoleInput.php]: PHP doesn't complain when passing `null` for those argument so it's a valid value.

Comment (sohelrana820) [reply src/Console/ConsoleInput.php]: @ADmad, so I need to revert this changes?

File: tests/TestCase/Filesystem/FileTest.php [hunk 5]
```diff
@@ -503,7 +503,7 @@ public function testLastChange()
     /**
      * testWrite method
      *
-     * @return void
+     * @return bool|void
```
Comment (dereuromark) [inline tests/TestCase/Filesystem/FileTest.php]: Same issue here. Must stay void and instead the test should be skipped properly.

**Pull request merged**